sig-cloud-provider/CHARTER.md

```diff
@@ -0,0 +1,100 @@
+* Issues impacting multiple subprojects in the SIG should be resolved by consensus of the owners of the involved subprojects; if a decision can’t be made, it should be escalated to the SIG Chairs
+
+## Roles
+The following roles are required for the SIG to function properly. In the event that any role is unfilled, the SIG will make a best effort to fill it. Any decisions reliant on a missing role will be postponed until the role is filled.
+
+
+### Chairs
+* 3 chairs are required
+* Run operations and processes governing the SIG
+* An initial set of chairs was established at the time the SIG was founded.
+* Chairs MAY decide to step down at anytime and propose a replacement, who must be approved by all of the other chairs. This SHOULD be supported by a majority of SIG Members.
+* Chairs MAY select additional chairs using lazy consensus amongst SIG Members.
+* Chairs MUST remain active in the role and are automatically removed from the position if they are unresponsive for > 3 months and MAY be removed by consensus of the other Chairs and members if not proactively working with other Chairs to fulfill responsibilities.
+* Chairs WILL be asked to step down if there is inappropriate behavior or code of conduct issues
+* SIG Cloud Provider cannot have more than one chair from any one company.
+
+### Subproject/Provider Owners
+* There should be at least 1 representative per subproject/provider (though 3 is recommended to avoid deadlock) as specified in the OWNERS file of each cloud provider repository.
+* MUST be an escalation point for technical discussions and decisions in the subproject/provider
+* MUST set milestone priorities or delegate this responsibility
+* MUST remain active in the role and are automatically removed from the position if they are unresponsive for > 3 months and MAY be removed by consensus of other subproject owners and Chairs if not proactively working with other Subproject Owners to fulfill responsibilities.
+* MAY decide to step down at anytime and propose a replacement. This can be done by updating the OWNERS file for any subprojects.
+* MAY select additional subproject owners by updating the OWNERs file.
+* WILL be asked to step down if there is inappropriate behavior or code of conduct issues
+
+### SIG Members
+
+Approvers and reviewers in the OWNERS file of all subprojects under SIG Cloud Provider.
+
+## Long Term Goals
+
+The long term goal of SIG Cloud Provider is to promote a vendor neutral ecosystem for our community.  Vendors wanting to support Kubernetes should feel equally empowered to do so
+as any of today’s existing cloud providers; but more importantly ensuring a high quality user experience across providers. The SIG will act as a central group for developing
+the Kubernetes project in a way that ensures all providers share common privileges and responsibilities. Below are some concrete goals on how SIG Cloud Provider plans to accomplish this.
+
```

Comment:
I would like us to add a section about advocacy that includes supporting provider working group meetings at events and other use of community resources. One of the large impacts this will have on the community is the potential loss of collaboration and meeting space at events.

SIG-OpenStack, for example, regularly has between 40-60 attendees at it's SIG meetings at KubeCon, and the loss of access to these resources will be one of the largest impacts our community will feel.

Reply:
We spoke about this on slack, we'll be addressing this during the transitional period :)